### PR TITLE
Drop opcache from cli actions

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: 7/apache/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN set -ex; \
 	; \
 	\
 	docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr; \
-	docker-php-ext-install -j "$(nproc)" \
+	docker-php-ext-install \
```
Comment (tianon) [inline 7/apache/Dockerfile]: Why are we dropping all the `-j` values?

Comment (J0WI) [reply 7/apache/Dockerfile]: For consistency with the `wordpress` image. Is there a reason why this defaults to `1`?
https://github.com/docker-library/php/blob/04e67dc6bb76783d2d1b64513b8f42961e578645/docker-php-ext-install#L47

Comment (tianon) [reply 7/apache/Dockerfile]: Ah, then maybe we should update WordPress.

It defaults to `make`'s default (we don't explicitly `-j1` anywhere), mostly to be considerate of `php` users' machines -- for the official images though, we want to make full use of our build systems as much as possible (especially chasing the dragon of potentially faster builds), which is why we typically use `-j "$(nproc)"` in ours.

Comment (J0WI) [reply 7/apache/Dockerfile]: I reverted this. I also noted that freetype support was missing in the Debian variant.

**Comment by yosifkit** [comment]
More changes than just dropping opcache, but it all looks sane to me.

Just noting that this change addresses #97 (which was closed shortly after opening).

**Pull request merged**